### PR TITLE
add support for renaming/moving files

### PR DESCRIPTION
**Pull request by witsch**
for now i've created a separate test class as i didn't really grok the reason for the extra `DummyDataManager`.  however, if the patch looks alright (and somebody tells me what the dummy manager is for :)), i'll be happy to add a `rename_file` there as well…

### CODE REVIEW

File: src/repoze/filesafe/__init__.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ def _remove_manager(*a):
     except AttributeError:
         pass
 
+
```
Comment (tseaver) [inline src/repoze/filesafe/__init__.py]: Please don't run a `pep8` tool on the sources when submitting a non-janitorial PR.

**Comment by tseaver** [comment]
`testing.DummyDataManager` exists to ease testing of dependent apps.  It should support the same API as `manager.FileSafeDataManager', but allow for easy introspection of the changes induced by the app's tests.

**Comment by witsch** [comment]
On 21 Jan 2015, at 21:03, Tres Seaver <notifications@github.com> wrote:
> `testing.DummyDataManager` exists to ease testing of dependent apps. It should support the same API as `manager.FileSafeDataManager', but allow for easy introspection of the changes induced by the app's tests.

ah, now it makes sense — thank you.  i’ve updated the dummy manager in 7e3f4772.

**Comment by witsch** [comment]
On 27 Jan 2015, at 11:51, Andreas Zeidler <az@zitc.de> wrote:
> ah, now it makes sense — thank you.  i’ve updated the dummy manager in 7e3f4772.

i guess i should have actually read the docs — i only grep’ed for `DummyDataManager`… :)

**Comment by tseaver** [comment]
@wichert LGTM.

**Comment by wichert** [comment]
@witsch can you sign the contributors file as well?

**Comment by wichert** [comment]
FYI this has now been released with version 2.2.

**Pull request merged**